src/animations/loading-dots-spinner/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface LoadingDotsSpinnerProps {
```

Comment:
I feel that this is going to be a generic prop for all lottie animations in future. Perhaps this could be made generic and added on a higher level instead.

Reply:
I have applied the changes fyi.